tao_compiler/.bazelrc

```diff
@@ -0,0 +1,8 @@
+startup --host_jvm_args=-Djdk.http.auth.tunneling.disabledSchemes=
+
+try-import ../tf_community/.bazelrc
+try-import ../tf_community/.tf_configure.bazelrc
+try-import %workspace%/.bazelrc.user
+
+build --disk_cache=~/.cache
```

Comment:
Do we need disk cache or remote cache？

Reply:
according to the experience of torch-blade, disc-cache is more stable.

Reply:
I think remote-cache is stable enough (tao compiler uses remote cache without this PR) and can be GC automatically while disc cache can not be GC. also CC @qiuxiafei @tanyokwok

Reply:
The TorchBlade's ci script already deleted the legacy disk cache(older than 10 days each time) for a while. I must be stable enough for TorchBlade. FYI https://github.com/alibaba/BladeDISC/pull/662

Reply:
We can GC the disc-cache data using this code snippet: https://github.com/alibaba/BladeDISC/blob/main/scripts/ci/test_pytorch_blade.sh#L28-L38

But I agree that does not change the Bazel cache behavior in this PR.